x-pack/plugins/apm/public/components/app/TransactionOverview/TransactionBreakdown/TransactionBreakdownKpiList.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import {
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiText,
+  EuiTitle
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { InfraFormatterType } from '../../../../../../infra/public/lib/lib';
+import { FORMATTERS } from '../../../../../../infra/public/utils/formatters';
+
+interface TransactionBreakdownKpi {
+  name: string;
+  percentage: number;
+  count: number;
+  color: string;
+}
+
+interface Props {
+  kpis: TransactionBreakdownKpi[];
+}
+
+const Dot = styled.span`
+  display: block;
+  width: 8px;
+  height: 8px;
+  border-radius: 4px;
+  background-color: ${props => props.color};
+`;
+
+const KpiDescription: React.FC<{
+  name: string;
+  color: string;
+}> = ({ name, color }) => {
+  return (
+    <EuiFlexGroup
+      alignItems="center"
+      gutterSize="s"
+      direction="row"
+      wrap={false}
+    >
+      <EuiFlexItem grow={false}>
+        <Dot color={color} />
+      </EuiFlexItem>
+      <EuiFlexItem>
```

Comment:
I reckon we need to add a min-width to this like: `<EuiFlexItem style={{ minWidth: 110 }}>`